css/state.css

```diff
@@ -0,0 +1,43 @@
+
+.banner li:nth-of-type(1):hover {
+  color: #3b5998;
+}
+
+.banner li:nth-of-type(2):hover {
+  color: #1dcaff;
+}
+
+.banner li:nth-of-type(3):hover {
+  color: red;
+}
+.banner li:nth-of-type(4):hover {
+  color: green;
+}
+.banner li:nth-of-type(5):hover {
+  color: darkblue;
+}
+
+.banner li:nth-of-type(6):hover {
+  color: darkred;
+}
+
+.banner li:nth-of-type(7):hover, .banner li:nth-of-type(8):hover {
+  color: purple;
+}
+
+input[type=checkbox]:checked + label{
+  text-decoration: line-through;
+}
+
+.recipe__author__button-like:hover {
+  color: white;
+  font-family: 'Patrick Hand', sans-serif;
+  text-align: center;
+  font-size: 30px;
```

Comment:
Remember you will likely also need to also declare a line-height in order for the height to not 'jump' between the two states.